From gcoady@gmail.com Sat Nov  5 19:44:12 2005
Message-ID: <436D45D3.4060807@gmail.com>
Date: Sun, 06 Nov 2005 10:52:51 +1100
From: Grant Coady <gcoady@gmail.com>
To: Greg KH <gregkh@suse.de>
Cc: Chun-Chung Chen <cjj@u.washington.edu>
Subject: [PATCH] pci_ids cleanup: fix two additional IDs in bt87x

From: Grant Coady <gcoady@gmail.com>

pci_ids cleanup: fixup bt87x.c: two macro defined IDs missed in prior cleanup.

Caught by Chun-Chung Chen <cjj@u.washington.edu>: "In the patch for bt87x.c,
you seemed have missed the two occurrences of BT_DEVICE on line 897 and
line 898."

Signed-off-by: Grant Coady <gcoady@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 sound/pci/bt87x.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- gregkh-2.6.orig/sound/pci/bt87x.c
+++ gregkh-2.6/sound/pci/bt87x.c
@@ -897,8 +897,8 @@ static void __devexit snd_bt87x_remove(s
 /* default entries for all Bt87x cards - it's not exported */
 /* driver_data is set to 0 to call detection */
 static struct pci_device_id snd_bt87x_default_ids[] = {
-	BT_DEVICE(878, PCI_ANY_ID, PCI_ANY_ID, 0),
-	BT_DEVICE(879, PCI_ANY_ID, PCI_ANY_ID, 0),
+	BT_DEVICE(PCI_DEVICE_ID_BROOKTREE_878, PCI_ANY_ID, PCI_ANY_ID, 0),
+	BT_DEVICE(PCI_DEVICE_ID_BROOKTREE_879, PCI_ANY_ID, PCI_ANY_ID, 0),
 	{ }
 };
 
